src/modules/job-manager/restart.c

```diff
@@ -266,8 +266,21 @@ int restart_from_kvs (struct job_manager *ctx)
                                 __FUNCTION__, (uintmax_t)job->id);
             }
         }
-        else if ((job->state & FLUX_JOB_STATE_RUNNING) != 0)
+        else if ((job->state & FLUX_JOB_STATE_RUNNING) != 0) {
             ctx->running_jobs++;
+            job->reattach = 1;
+            if ((job->flags & FLUX_JOB_DEBUG)) {
+                if (event_job_post_pack (ctx->event,
+                                         job,
+                                         "debug.flux-reattach",
```

Comment:
why is "flux" in the name?

At the moment I'm not finding  the event that signifies completion but if it's still there, prob should balance the names, like `debug.exec-reattach-start` and `debug.exec-reattach-finish`.  (those are the event suffixes used by prolog/epilog events)

Reply:
good point, i was just following the `flux-restart` event from earlier.  start and finish seem better.